### PR TITLE
Add major version updater workflow

### LINKED ISSUE #26: Major version is not tagged
Most actions, like [`checkout`](https://github.com/actions/checkout) publish two tags for each release:
- A new semver version, e.g. `v3.5.2`
- A force-pushed major semver version, e.g. `v3`

This strategy allows people to reference a major version in their workflow so they get all non-breaking updates without having to manually update or work through dependabot/renovate PRs, e.g.:

For example, right now you must do something like this for `stale-repos`:
```yaml
 uses: github/stale-repos@v1.1.0
```

When a new version is released, it would need to be updated everywhere it is used to get the latest and greatest. Moving to a release that includes an override of the current major version would allow people to get free updates, and would look like this:

```yaml
 uses: github/stale-repos@v1
```

_note:_ The examples above use the normal github repo checkout syntax instead of the `docker://` syntax. Both of them work equally as well, but I think the repo checkout is more intuitive/common.

### PR DESCRIPTION
**Pull request by ahatzz11**
This PR adds support for creating/overriding a major version when a release is made. This allows people to have a single reference that does not need to be updated in order to get new non-breaking stuff.

Before:
```yaml
 uses: github/stale-repos@v1.1.0
```

After:
```yaml
 uses: github/stale-repos@v1
```

Fixes #26

### CODE REVIEW

File: .github/workflows/major-version-updater.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        id: version
+        run: |
+          tag=${GITHUB_REF/refs\/tags\//}
+          version=${tag#v}
+          major=${version%%.*}
+          {
+            echo tag=${tag}
+            echo "version=${version}
+            echo "major=${major}" 
+          } >> "$GITHUB_OUTPUT"
```
Comment (ahatzz11) [inline .github/workflows/major-version-updater.yml:24]: @zkoppert The linter forced this syntax, but I personally think this is a case of 'linter makes things less-common and more annoying', vs:

```bash
echo "tag=${tag}" >> "$GITHUB_OUTPUT"
echo "version=${version}" >> "$GITHUB_OUTPUT"
echo "major=${major}" >> "$GITHUB_OUTPUT"
```

Happy to keep it as is but thought I'd call it out.

Comment (zkoppert) [reply .github/workflows/major-version-updater.yml]: I agree with you here, I'm disabling that check and reverting to original syntax...

Comment (ahatzz11) [reply .github/workflows/major-version-updater.yml]: Nice! Appreciate the change 👍

Comment (zkoppert) [reply .github/workflows/major-version-updater.yml]: Looks like I might not have the rc file in the correct place but i'll get there. 😅

**Pull request merged**